coordinator/package.json

```diff
@@ -55,5 +57,8 @@
     "ts-jest": "^26.1.0",
     "ts-node-dev": "^1.0.0-pre.44",
     "typescript": "^3.9.5"
+  },
+  "_moduleAliases": {
+    "@models": "models/library"
```

Comment:
runtime path aliases as ts transpiling does not cover this, see note with link in coordinator/src/app.ts, suggested approach is to use a pre-existing module loader, which we could look at including in future, for now this resolves runtime module loading with a couple of lines of code